JavaScript/amazon_reviews_scrapper/README.md

```diff
@@ -0,0 +1,11 @@
+# Amazon review scrapper
+
+A javascript to scrape reviews from Amazon.
+
+## Setup
+1. Clone the folder.  
+2. we will be using **cheerio** for this which you can install by using command ```npm install cheerio```. 
```

Comment:
Format the README properly. Use a tool like `Grammarly` for better formatting the sentences.